### ISSUE TITLE
Difficulty maintaining aligned crosshair

### ISSUE THREAD

**Issue by vndmtrx** [traverseda/PyCraft#33]
By moving the mouse, the algorithm that makes the camera drive mapping seems to have excessive acceleration when the camera is on the horizon height causing the mouse stick it in the zenith or nadir view.

**Comment by traverseda** [comment]
I can't replicate this. Would you mind sharing some information about your system?

It sounds like the sort of issue I've had with WINE when it's not capturing the mouse properly. Try pressing escape to release your mouse, then clicking the window to re-capture it.

**Comment by vndmtrx** [comment]
I reproduce it on VirtualBox and recorded the behaviour on a youtube video.

https://youtu.be/ZGcraBEuOiY

**Comment by traverseda** [comment]
Hmm, does it happen outside of virtualbox? My suspicion is that it could be how VB captures the cursor.

Anyone else with that linux want to chime in?

**Comment by vndmtrx** [comment]
Yes, It happened on my enviroment as well. I only used the VM to record the video. I check the mouse acceleration as well but it seems to not affect the problem so much.
